src/CMakeLists.txt

```diff
@@ -47,9 +47,9 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
     set(SERVICE_INSTALL_DIR "/etc/systemd/system")
     set(DATA_INSTALL_DIR "/var/lib/wazuh-agent")
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
-    set(BIN_INSTALL_DIR "C:/Program Files/wazuh-agent/bin")
-    set(CONFIG_INSTALL_DIR "C:/Program Files/wazuh-agent/config")
-    set(DATA_INSTALL_DIR "C:/Program Files/wazuh-agent/var")
+    set(BIN_INSTALL_DIR "C:\\Program Files\\wazuh-agent")
+    set(CONFIG_INSTALL_DIR "C:\\ProgramData\\wazuh-agent\\config")
+    set(DATA_INSTALL_DIR "C:\\ProgramData\\wazuh-agent\\var")
```

Comment:
Should we be hardcoding these paths? There is $ENV{ProgramFiles}  and $ENV{ProgramData} we might want to try for this.

Reply:
Done!